File: dayThree/afternoonSession/Python/mini_parking_system.py
car_slots = [0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]

def checking_a_slot(car_slots, slot_number):

    if car_slots[slot_number] == 0:
        return "Empty"
    elif car_slots[slot_number] == 1:
        return "Occupied"

def parking_a_car(car_slots, slot_number):
    checking_slot = checking_a_slot(car_slots, slot_number)

    if checking_slot == "Empty":
        car_slots[slot_number] = 1
        return "Now Occupied"
    else:
        return "Already Occupied Before Now"


def unparking_a_car(car_slots, slot_number):
    parking_car = parking_a_car(car_slots, slot_number)

    if parking_car == "Now Occupied":
        car_slots[slot_number] = 0
        return "Car Unparked"
    elif parking_car == "Already Occupied Before Now":
        return "Car Not Yours"

File: dayThree/afternoonSession/Java/MiniParkingSystem.java
public class MiniParkingSystem{
    public static String checkingASlot(int[] carSlots, int slotNumber){
        if(carSlots[slotNumber] == 0){
            return "Empty";
        }
        else if(carSlots[slotNumber] == 1){
            return "Occupied";
        }

    }

    public static String parkingACar(int[] carSlots, int slotNumber){
        String checkingSlot = checkingASlot(carSlots, slotNumber);

        if(checkingSlot == "Empty"){
            carSlot[slotNumber] = 1;
            return "Now Occupied";
        }
        else{
            return "Already Occupied Before Now";
        }
    }

    public static String unparkingACar(int[] carSlots, int slotNUmber){
        String parkingCar = parkingACar(carSlots, slotNumber);

        if(parkingCar == "Now Occupied"){
            carSlot[slotNumber] = 0;
            return "Car Unparked";
        }
        else if(parkingCar == "Already Occupied Before Now"){
            return "Car Not Yours"
        }

    }

}
